File: app/src/main/java/com/u1fukui/bbs/utils/StringUtils.java
package com.u1fukui.bbs.utils;


public class StringUtils {

    private StringUtils() {}

    public static boolean isBlank(String text) {
        if (text == null) {
            return true;
        }
        return text.trim().isEmpty();
    }

    public static int isLength(String text) {
        if (text == null) {
            return 0;
        }
        return text.length();
    }
}

File: app/src/main/java/com/u1fukui/bbs/utils/StringUtils.kt
package com.u1fukui.bbs.utils


object StringUtils {

    fun isBlank(text: String?): Boolean {
        return text?.trim { it <= ' ' }?.isEmpty() ?: true
    }

    fun isLength(text: String?): Int {
        return text?.length ?: 0
    }
}
